browser/js/auth/auth.js

```diff
@@ -0,0 +1,28 @@
+app.controller('AuthCtrl', function ($scope, AuthService, $state, $uibModalInstance) {
+
+    $scope.login = {};
+    $scope.error = null;
+
+    $scope.sendAuth = function (authInfo) {
+
+        $scope.error = null;
+
+        AuthService.login(authInfo)
+            .then(null, function () {
+                return AuthService.signup(authInfo);
+            })
+            .then(function () {
+                $uibModalInstance.close();
+
+                //check if user.requiresPasswordReset
+                //go to change password page
+                // if (user.requiresPasswordReset) $state.go('user');
```

Comment:
We can! I had just added that as comments earlier before we had the password reset setup. I will go back and add it.